### DISCUSSION TITLE
issue: React-hook-form focusing aren't working  with SyncFusion React

### DISCUSSION THREAD

**Discussion by atillaaliyev** [react-hook-form/react-hook-form#8658] [Q&A] [answered]
### Version Number

7.33.1

### Codesandbox/Expo snack

https://codesandbox.io/s/react-hook-form-with-syncfusion-lb67ft

### Steps to reproduce

1. Go to sandbox link
2. Click on validate button

### Expected behaviour

When the form has any error, it should focus first item that has error.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by bluebill1049** [comment]
This is not an issue with hook form as the forward ref is not the actual input reference.
<img width="1038" alt="Screen Shot 2022-07-09 at 9 16 33 pm" src="https://user-images.githubusercontent.com/10513364/178103339-9bc9d1fc-3494-46c1-949f-81e7d6c1a978.png">

**Comment by atillaaliyev** [comment]
Thank you, I guessed it , but i wanted sure.

**Comment by Moshyfawn** [comment, marked as answer]
@atillaaliyev, you can get the underling input element of a SyncFusion component by accessing `ref?.element` and passing it into the RHF's `ref` callback like this
```js
<Controller
  render={({
    field
  }) => (
      <TextBoxComponent
        ref={(ref) => {
          field.ref(ref?.element)
        }}
      />
  )}
/>
```

**Comment by atillaaliyev** [comment]
Perfect answer, it worked. Thank u.
